rolling-shutter/snapshot/handler.go

```diff
@@ -0,0 +1,154 @@
+package snapshot
```

Comment:
@ulope I created this whole message handler module to mirror the other message handling code in the main branch. I have _not_ run any tests to confirm, that it actually works as intended.